sdk/identity/identity/test/manual-integration/AzureFunctions/IdentityTest/host.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": "2.0",
+  "logging": {
+    "applicationInsights": {
```

Comment:
Let's see if we can simplify here - Can we turn off app insights?

Reply:
You get a bunch of freebies if you use AppInsights and I don't think you have to do anything. I would just leave it in unless you think it's going to confuse people.